server/indexer/cesium_math.go

```diff
@@ -0,0 +1,43 @@
+package indexer
+
+import (
+	"math"
+)
+
+const (
+	_EPSILON1           = 0.1
+	_EPSILON12          = 0.000000000001
+	_EPSILON14          = 0.00000000000001
+)
+
+const (
+	_PI     = math.Pi
+	_TWO_PI = 2 * math.Pi
+	_DEGREES_PER_RADIAN = 180.0 / math.Pi
```

Comment:
```suggestion
	Epsilon1           = 0.1
	Epsilon12          = 0.000000000001
	Epsilon14          = 0.00000000000001
	Pi     = math.Pi
	TwoPi = 2 * math.Pi
	DegreesPerRadian = 180.0 / math.Pi
```

(Note that this suggestion is handwritten and not formatted)

Adding an underscore at the beginning is not particularly meaningful in Golang and is not very common. Golang rarely uses `snake_case`. Use `camelCase` (private) or `CamelCase` (public). In this case CamelCase is ok.